resources/lang/en/language_names.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+# modified at 23-01-2018 13:48
+
+return [
+    'en' => 'English',
```

Comment:
I think that seeder also includes German and French languages

Reply:
yes, they are seeded in migrations... fix will come